conf/rak_rak7391/files/etc/uci-defaults/02_rtc

```diff
@@ -0,0 +1,7 @@
+. /lib/functions/uci-defaults.sh
+. /lib/functions.sh
+. /lib/functions/system.sh
+
+sed "s|exit 0|echo pcf8563 0x51 > /sys/bus/i2c/devices/i2c-1/new_device\nexit 0|" -i /etc/rc.local 
```

Comment:
Would it make sense to add a `files/etc/rc.local` file directly?

Reply:
It would be simpler, of course. Any risk in overwriting the default `rc.local` file?

Reply:
I believe by default it is empty, so no :)